types/office-js-preview/index.d.ts

```diff
@@ -476,6 +516,114 @@ declare namespace Office {
         */
         value: T;
     }
+    /**
+     * Message used in the visibility mode changed event.
```

Comment:
Later, you use "VisbilityModeChanged event" to describe this. I like using the actual name. Can we surround it with code ticks (`)?

Reply:
Done